giskard/ml_worker/ml_worker.py

```diff
@@ -30,16 +33,24 @@ async def _start_grpc_server(is_server=False):
     return server, port
 
 
-async def start_ml_worker(is_server=False, remote_host=None, remote_port=None):
+async def start_ml_worker(is_server=False, is_silent=False,remote_host=None, remote_port=None, token=None):
     from giskard.ml_worker.bridge.ml_worker_bridge import MLWorkerBridge
-
+    url = f'{remote_host}/api/v2/settings/general'
+    host_name = urlparse(remote_host).hostname
+    res = requests.get(url, headers={"Authorization": f"Bearer {token}"})
+    if res.status_code == 401:
```

Comment:
After this check we have to verify that the status code is actually 200 and raise an exception if it's not like "Failed to connect to Giskard instance"